.github/ISSUE_TEMPLATE/1-tasks.md

```diff
@@ -13,10 +13,7 @@ started:
 actual:
 completed:
 confirmed:
-dependencies:
-- [ ]
-supersedes:
--
+
```

Comment:
Would users have to remember `dependencies:` is a valid field?

Reply:
Yes, but I think that's easier than remembering to delete it every time if there are no dependencies.

Reply:
Probably better to just delete supersedes then, as dependencies are common. Which would fix the rendering issues as it mainly affects the section after rendering due to the empty text